### PR TITLE
Add support for termux

### PR DESCRIPTION
**Pull request by Freed-Wu**
Screenshot-20210310180229-1366x768

`termux-api` maybe hang in some conditions.
so wait <https://github.com/termux/termux-api/issues/402> to be fixed may be
better.

### CODE REVIEW

**Comment by Freed-Wu** [comment]
i forgot to write `Fix #9` in the commit message... Sorry!

**Comment by lambdalisue** [comment]
I'll check it in this weekend. Ping me if I've not replied even after the weekend.

File: autoload/battery/backend/termux.vim [hunk 1]
```diff
@@ -0,0 +1,19 @@
+" Ref: https://wiki.termux.com/wiki/Termux-battery-status
+let s:Job = vital#battery#import('System.Job')
+
+function! s:termux_update() abort dict
+  let self.is_charging = json_decode(system('termux-battery-status')).status !=# 'NOT_CHARGING'
+  let self.value = json_decode(system('termux-battery-status')).percentage + 0
+endfunction
```
Comment (lambdalisue) [inline autoload/battery/backend/termux.vim:24]: `system()` should not be used. Use `System.Job` instead.

You can find the documentation here
https://github.com/lambdalisue/vital-Whisky/blob/master/doc/Vital/System/Job.txt

**Review by lambdalisue — CHANGES_REQUESTED** [review]

**Comment by Freed-Wu** [comment]
Screenshot from 2021-03-14 21-40-31

you are right. now it can work.

**Review by lambdalisue — APPROVED** [review]

**Comment by lambdalisue** [comment]
👍

**Pull request merged**